package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "standard | snazzy && npm run lint:typescript",
     "lint:fix": "standard --fix",
     "lint:typescript": "standard --parser @typescript-eslint/parser --plugin @typescript-eslint/eslint-plugin ./*.ts",
-    "test": "standard | snazzy && tap test/*.js && npm run typescript",
+    "test": "NODE_OPTIONS=--experimental-modules standard | snazzy && tap test/*.js && npm run typescript",
```

Comment:
```suggestion
    "test": "NODE_OPTIONS=--experimental-modules standard | snazzy && tap --no-esm test/*.js && npm run typescript",
```

Reply:
What is the reason for this change?
With this change 5 tests would fail.

Reply:
The reason is that you are not targeting the Node experimental modules, but rather the `esm` implementation. They are going to be significantly different.